src/Linkify.tsx

```diff
@@ -149,6 +149,7 @@ export const sanitizeHtmlParams: IExtendedSanitizeOptions = {
     allowedTags: [
         "font", // custom to matrix for IRC-style font coloring
```

Comment:
Is this list of tags the same as that in the spec? It might be nice to link to the spec if so